Dm supports google cloud (#15381)
pingcap/docs: tidb-cloud/tidb-cloud-release-notes.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

## November 21, 2023

**General changes**

> - TiDB Cloud Data Migration now supports high-speed physical mode in Google Cloud.

Author: ```suggestion
- [Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md) supports high-speed physical mode for TiDB clusters deployed on Google Cloud.
```

>     The migration speed of physical mode can reach up to 110 MiB/s, which is 2.4 times faster than logical mode. The improved performance is suitable for quickly migrating a large volume of data to TiDB Cloud. Now you can use physical mode for TiDB clusters deployed on AWS and Google Cloud.

Author: ```suggestion
    Now you can use physical mode for TiDB clusters deployed on AWS and Google Cloud. The migration speed of physical mode can reach up to 110 MiB/s, which is 2.4 times faster than logical mode. The improved performance is suitable for quickly migrating large datasets to TiDB Cloud.
```
